tests/rpc/rpc_05-state_test.go

```diff
@@ -161,8 +161,9 @@ func TestStateRPCResponseValidation(t *testing.T) { //nolint:tparallel
 	})
 }
 
+// TODO: add test against latest gssmr runtime
```

Comment:
```suggestion
// TODO: add test against latest dev runtime
// See https://github.com/ChainSafe/gossamer/issues/2705
```

Reply:
This is still `gssmr`, since we import the `gssmr` runtime in this tests.  We can look into the reason why we do this for this tests when we revisit in #2705